pkg/ccl/backupccl/backup.proto

```diff
@@ -166,3 +166,18 @@ message RestoreProgress {
   int64 progressIdx = 2;
   roachpb.Span dataSpan = 3 [(gogoproto.nullable) = false];
 }
+
+message BackupPlanningTraceEvent {
+  message Spans {
+    repeated roachpb.Span spans = 1 [(gogoproto.nullable) = false];
+  }
+  map<int32, Spans> node_to_spans = 1 [(gogoproto.nullable) = false];
```

Comment:
Yeah crossed my mind while I was adding this proto. I don't have a very good intuition about what serializing/deserializing an O(ranges) proto costs us. I believe we will need to serialize/deserialize if we pull the trace from a node other than the coordinator node, as well as when we use the existing `payloads_for_span` generator - https://github.com/cockroachdb/cockroach/blob/723ded97a32a7771ddc5b34fa858b3f60374389a/pkg/sql/sem/builtins/generator_builtins.go#L1552.

I thought it would be valuable to know which span is being executed on where, but maybe we can get away with adding the span to the trace at the ExportRequest level, when the node actually serves the export request?